zetaclient/chains/solana/signer/outbound_tracker_reporter.go

```diff
@@ -27,11 +28,19 @@ func (signer *Signer) reportToOutboundTracker(
 	txSig solana.Signature,
 	logger zerolog.Logger,
 ) {
+	// prepare logger
+	logger = logger.With().
+		Str(logs.FieldMethod, "reportToOutboundTracker").
+		Int64(logs.FieldChain, chainID).
+		Uint64(logs.FieldNonce, nonce).
+		Str(logs.FieldTx, txSig.String()).
+		Logger()
+
 	// set being reported flag to avoid duplicate reporting
 	alreadySet := signer.Signer.SetBeingReportedFlag(txSig.String())
 	if alreadySet {
 		logger.Info().
-			Msgf("reportToOutboundTracker: outbound %s for chain %d nonce %d is being reported", txSig, chainID, nonce)
+			Msgf("outbound %s for chain %d nonce %d is being reported", txSig, chainID, nonce)
```

Comment:
- no need to log chain id as we keep this in `logger`
- consider moving txSig and nonce to log fields

Reply:
The only reason that I kept them is to make these fields visible in Datadog, so people can see them (they're important fields) without having to mouse click on the log. But I'll ask Devops to see if Datadog can be configured to show the fields.

Reply:
The Devops team would configure Datadog to show above log fields. After a chat with @CharlieMc0 
I removed the repeated log fields from the log message in commit https://github.com/zeta-chain/node/pull/2628/commits/d659e5ae060161a26f38b10146f4ef3a96aecd48